usaon_vta_survey/templates/macros/javascript_delete.j2

```diff
@@ -0,0 +1,8 @@
+{% macro add_javascript__delete() -%}
+  <script>
+    function remove_relationship(url, redirect_url) {
```

Comment:
I don't think we should let the view pass in the `redirect_url`. We are already getting that information in the response from the fetch request! In order to access this information, you need to chain the `fetch` promise:

```javascript
      fetch(url, { method: 'DELETE'})
        .then(response => { 
          // redirect to the URL provided in the 303 response
          // this information may be in `response.headers`?
          // https://developer.mozilla.org/en-US/docs/Web/API/Response
        })
        .catch(error => console.log(error));
```

It's also really important to capture any errors, too!